src/redux/modules/filesToValidate.js

```diff
@@ -10,6 +10,7 @@ import {FilesApi} from '../../api/files';
 export const filesToValidateActions = {
     delFile: createAction('FILE_TO_VALIDATE_DEL'),
     addFile: createAction('FILE_TO_VALIDATE_ADD_REQUEST', file => file),
+    addLocalFile: createAction('LOCAL_FILE_TO_VALIDATE_ADD_REQUEST', file => file.path),
```

Comment:
I'd rather avoid duplicating the code for electron and non-electron environment.
Instead I propose that we hide all the differences in API modules.

So, let's extend `FilesApi.uploadFile` method so that it checked the environment and did corresponding call. The response is going be the same for 'LOCAL' and 'REMOTE' file, so there will be no need to duplicate epics|reducers etc.